guides/common/assembly_importing-content.adoc

```diff
@@ -71,6 +71,14 @@ include::modules/proc_adding-custom-rpm-repositories-el.adoc[leveloffset=+1]
 :yum_repo_extras_url: https://yum.oracle.com/repo/OracleLinux/OL{os_major}/addons/x86_64/
 include::modules/proc_adding-custom-rpm-repositories-el.adoc[leveloffset=+1]
 
+include::modules/proc_creating-uln-repositories.adoc[leveloffset=+1]
+
+include::modules/snip_uln-repositories-for-oracle-linux-9.adoc[]
+
+include::modules/snip_uln-repositories-for-oracle-linux-8.adoc[]
+
+include::modules/snip_limit-download-concurrency-for-uln-content.adoc[]
```

Comment:
I'd add this snippet inside the procedure perhaps. Non-blocking.

Reply:
Applied.